src/cmd/src/bin/greptime.rs

```diff
@@ -235,3 +235,29 @@ async fn main() -> Result<()> {
 
     Ok(())
 }
+
+/// Default path to store configuration
+const DEFAULT_OPTIONS_PATH: &str = "/tmp/greptimedb/conf/opts.toml";
+
+fn write_opts_into_tmp_dir(opts: &Options) {
+    let opts_toml = toml::to_string(&opts).unwrap();
+
+    if let Some(parent_dir) = std::path::Path::new(DEFAULT_OPTIONS_PATH).parent() {
```

Comment:
path could be taken from the env variable and `DEFAULT_OPTIONS_PATH` could be used if a variable is not specified